client/my-sites/marketplace/components/progressbar/index.tsx

```diff
@@ -1,18 +1,7 @@
-import { ProgressBar } from '@automattic/components';
-import styled from '@emotion/styled';
-import { TranslateResult, useTranslate } from 'i18n-calypso';
+import { TranslateResult } from 'i18n-calypso';
 import { useEffect, useState } from 'react';
-
-const Container = styled.div`
-	text-align: center;
-`;
-const StyledProgressBar = styled( ProgressBar )`
-	margin: 20px 0;
-`;
-
-const Title = styled.h1`
-	font-size: 2rem;
-`;
+import './style.scss';
+import Loading from 'calypso/components/loading';
```

Comment:
Nit: style imports are usually after all js imports. Ideally, this would be caught by linting.

```suggestion
import Loading from 'calypso/components/loading';
import './style.scss';
```